typescript/ws-protocol/README.md

```diff
@@ -154,9 +120,30 @@ async function main() {
 main().catch(console.error);
 ```
 
+Copy the template code into a file (e.g. `client.js`) and start up a [Foxglove Websocket server](#server-template). In a separate terminal window, run the client code (e.g. `node client.js`).
+
+You should see the following output if both your server and client are running correctly:
+
+```
+$ node client.js
```

Comment:
It occurs to me that we should update the `simple-client` script to handle JSON messages as well as protobuf, since JSON is easier and it's a bit weird to have one example that supports only protobuf and one that supports only JSON. Doesn't need to be done in this PR.

Reply:
Agreed – the long-term ideal is probably for the example and implementation packages to cover all supported encodings. So something like:

- `foxglove-websocket-examples` - Python examples package for JSON (existing) and Protobuf (TODO)
- `foxglove-websocket` – Python implementation package without any encoding dependencies (i.e. `json`; TODO)
-  `@foxglove/ws-protocol-examples` - TS examples package for JSON (TODO) and Protobuf (existing)
- `@foxglove/ws-protocol` - TS implementation package without any encoding dependencies (i.e. `ws`; TODO)

Reply:
I went down a rabbit hole today learning about python optional dependencies. The tl;dr is that as of #29 you can `pip install foxglove-websocket[examples]` which will bring in the optional protobuf dependency.

Reply:
I guess similarly to the server template above, this client can be moved into the example scripts package and removed from the readme (in a separate PR)